boxes/edges.py

```diff
@@ -1461,12 +1461,15 @@ class ChestHingeSettings(Settings):
 
  * pin_height : 2.0 : radius of the disc rotating in the hinge (multiples of thickness)
  * hinge_strength : 1.0 : thickness of the arc holding the pin in place (multiples of thickness)
+ * finger_joints_on_{box,lid} : False : whether to include finger joints on the edge with the box or lid
```

Comment:
Please make this two separate lines as they are used to generate the descriptions in the UI.

For now there is no way to have hidden parameters that ar enot shown in the UI but are set only by the generators. But this is not the first place where such thing is useful and I might add that feature at some point in the future. Probably somehthing like parameters starting with `_` being hidden.

As they for now can be deselected by the user you can either not render the wall with the hex holes for the side that got unchecked or may be force the setting to `True` in the generator code.

Reply:
Thanks, fixed